src/cli/moz-cache.js

```diff
@@ -1,5 +1,6 @@
 import { Scheduler } from 'taskcluster-client';
 import slugid from 'slugid';
+import createHash from '../hash';
```

Comment:
This is an ignorant javascript question: Is the reason to do that so that we don't import `crypto` multiple times?